### PR TITLE
Theme-related adjustments

### PR DESCRIPTION
**Pull request by elanals**
Details in ticket [here](https://www.notion.so/datawrapper/Refactor-migrate-themes-f5999346ac864c019d75e0d81f6db8dc).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
     "name": "@datawrapper/chart-core",
+<<<<<<< HEAD
+    "version": "8.3.1",
+=======
     "version": "8.4.0",
+>>>>>>> master
```
Comment (sto3psl) [inline package-lock.json:7]: just a heads up. That was a merge conflict at some point :)

Comment (elanals) [reply package-lock.json:7]: oopsie thought i'd resolved that. This branch has been a bit confusing, don't know how many times i've merged, must have slipped through the net at some point. Thanks!!

**Comment by gka** [comment]
@elanals I think everything from this by now added in other PRs, so this could be closed now. But you might want to take another look that I didn't forget anything?

**Comment by elanals** [comment]
We can close this. I'll pull master and check if anything is missing and bring it up in another PR if so

**Pull request closed without merging**